vault/resource_aws_secret_backend_test.go

```diff
@@ -76,6 +76,33 @@ func TestAccAWSSecretBackend_basic(t *testing.T) {
 	})
 }
 
+func TestAccAWSSecretBackend_fallback(t *testing.T) {
```

Comment:
Nice! Could we also add an update step and an import step to complete out the full testing loop?

Reply:
Added an import and update test.